package.json

```diff
@@ -154,6 +154,7 @@
     "mnemonist": "0.38.1",
     "multiformats": "9.6.2",
     "nanoid": "3.2.0",
+    "p-queue": "7.2.0",
```

Comment:
dont want to block this PR and this dep seems safe, but it's strictly needed?

Reply:
hmm i could hypothetically use setTimeout but this way is more effective

Reply:
yeah, think it should be okay, defs dont wanna implement queueing logic from scratch 😝